components/overledger/overledger.app.mjs

```diff
@@ -11,35 +29,55 @@ export default {
     },
   },
   methods: {
-    _baseUrl() {
-      return "https://api.overledger.io";
-    },
     _headers() {
       return {
         "Authorization": `Bearer ${this.$auth.oauth_access_token}`,
         "Content-Type": "application/json",
         "API-Version": "3.0.0",
       };
     },
+    _getBaseUrl() { //conditional for environment url selection.
+      return this.environment === "sandbox"
+        ? "https://api.sandbox.overledger.io"
+        : "https://api.overledger.io";
+    },
```

Comment:
The method will receive the environment selected by the user
```suggestion
    _getBaseUrl(environment) { //conditional for environment url selection.
      return environment === "sandbox"
        ? "https://api.sandbox.overledger.io"
        : "https://api.overledger.io";
    },
```

Reply:
change has been made